tools/serverpod_cli/lib/src/generator/dart/library_generators/class_generators/repository_classes.dart

```diff
@@ -307,6 +312,8 @@ class BuildRepositoryClass {
     Iterable<SerializableModelFieldDefinition> objectRelationFields,
   ) {
     return Method((m) => m
+      ..docs.add(
+          '/// Find a single [$className] from a table, using the provided [where]')
```

Comment:
Depending on what we decide to do with `find` we should copy the same comment over (+ adjustments for the single return value of course).